src/types/vendor/ethereumjs-wallet.d.ts

```diff
@@ -1,26 +1,15 @@
 /**
  * Incomplete -- Provider-engine needs its own typings
  */
-declare module 'ethereumjs-wallet/provider-engine' {
-  import { IPublicKeyOnlyWallet, IFullWallet } from 'ethereumjs-wallet';
-  class WalletSubprovider {
-    constructor(wallet: IPublicKeyOnlyWallet | IFullWallet, opts?: object);
-  }
-}
 
 declare module 'ethereumjs-wallet/thirdparty' {
```

Comment:
Is the new version copied from `ethereumjs-wallet`?
Do you know if 0.6.5 includes the type changes, if so we can try to remove the declaration file.

Reply:
It is not copied from `ethereumjs-wallet`, and `0.6.5` does not have type information. I've just modified what we had to fit the new version.